packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -295,6 +296,10 @@ export default class BuySellStore extends BaseStore {
         this.setShowAdvertiserPage(false);
     }
 
+    hidePopup() {
+        this.should_show_popup = false;
+    }
```

Comment:
Need an action to just set `should_show_popup` to false without re-rendering/calling the list of adverts since current `setShouldShowPopup` is calling `getAdvertiserAdverts` when the popup is false